### ISSUE TITLE
GeoJSON LineString (a closed loop) display broken at zoom levels above 18

### ISSUE THREAD

**Issue by khatribharat** [mapbox/mapbox-gl-js#4202]
**Bug Demo** : [jsbin link](https://jsbin.com/hevojewaqo/edit?html,output)
**mapbox-gl-js version**: : 0.32.1

### Steps to Trigger Behavior
If you zoom out, the `LineString` displays properly, zooming in to levels above `18` breaks `LineString` display. This only occurs with `LineString`s which are closed loops.

### Expected Behavior
The `LineString` should display at all zoom levels

### Actual Behavior
The `LineString` is broken at all zoom levels above `18`

**Comment by ryanbaumann** [comment]
@khatribharat Looks like your linestring crosses a tile boundary at z18.  Feojson sources default to cut tiles at a maxzoom of 18 with geojson-vt.  Increasing this value to 21 renders properly.

https://jsbin.com/tulamelezo/edit?html,output
```
map.addSource("data", {
	"type": "geojson",
     "maxzoom" : 21,
	"data": {
```

**Comment by lbud** [comment]
I believe this was fixed between v0.33.1 and v0.34.0 (likely https://github.com/mapbox/mapbox-gl-js/pull/4413) — confirmed by switching Mapbox GL JS version in the fiddle provided above. Please open a new issue if you see this again.
